.github/workflows/python-package.yml

```diff
@@ -48,4 +48,7 @@ jobs:
             exit 1
         fi
         ./scripts/unittests.sh
+    - uses: trailofbits/gh-action-pip-audit@v1.0.0
```

Comment:
Would we want to upload the generated SBOMs into SBOM hub? Or is that out of scope for this PR?

Reply:
unfortunately its 1.4 which we dont currently support...

Reply:
ah alright.